Sources/ArcGISToolkit/Components/Augmented Reality/FlyoverSceneView.swift

```diff
@@ -187,3 +187,13 @@ private class ObservableARSession: NSObject, ObservableObject, ARSessionDelegate
         currentFrame = frame
     }
 }
+
+extension FlyoverSceneView {
+    func getCameraController() -> CameraController {
+        return cameraController
+    }
+    
+    func getInterfaceOrientation() -> InterfaceOrientation? {
+        return interfaceOrientation
+    }
+}
```

Comment:
we can just make the state properties internal

Reply:
Are we okay to forgo this recommendation from the `@State` docs?

> Declare state as private to prevent setting it in a memberwise initializer, which can conflict with the storage management that SwiftUI provides. [Source](https://developer.apple.com/documentation/swiftui/state#:~:text=an%20initial%20value.-,Declare%20state%20as%20private%20to%20prevent%20setting%20it%20in%20a%20memberwise%20initializer%2C%20which%20can%20conflict%20with%20the%20storage%20management%20that%20SwiftUI%20provides,-%3A)

Reply:
I'm ok with it if we aren't setting it in a memberwise init. Otherwise I'm ok with not testing those two properties as I don't think it's worth adding a function to workaround it.

Reply:
I'm ok either way

Reply:
The properties are now internal since they are not set by the init, however, a better option could be to not test the state properties at all and create an `initialCamera

Reply:
I'm ok not testing them if we feel we don't want to violate that Swift convention.